src/main/java/edu/harvard/iq/dataverse/search/savedsearch/SavedSearchServiceBean.java

```diff
@@ -101,11 +110,15 @@ public SavedSearch add(SavedSearch toPersist) {
         return persisted;
     }
 
-    public boolean delete(long id) {
+    public boolean delete(long id, boolean unlink) throws SearchException, CommandException {
         SavedSearch doomed = find(id);
         boolean wasDeleted = false;
         if (doomed != null) {
             System.out.println("deleting saved search id " + doomed.getId());
```

Comment:
While we're in this part of the code, should we delete this println? Or maybe convert it to our regular logger and set the level to "fine"?